src/state/fetch-backend.js

```diff
@@ -1,6 +1,6 @@
 import { useEffect, useState } from "react";
 
-export const fetchBackend = uname => {
+const FetchBackend = uname => {
```

Comment:
if this is a hook, it should be named `useFetchBackend`

Uppercase names are reserved to React Components

Reply:
I've updated this so that FetchBackend is a component, rather than a custom hook. Let me know if I should change it back (and then change the syntax)